packages/dds/merge-tree/api-report/merge-tree.legacy.alpha.api.md

```diff
@@ -390,20 +267,6 @@ export interface IMergeTreeObliterateSidedMsg extends IMergeTreeDelta {
 // @alpha (undocumented)
 export type IMergeTreeOp = IMergeTreeDeltaOp | IMergeTreeGroupMsg;
 
-// @alpha (undocumented)
-export interface IMergeTreeOptions {
-    attribution?: IMergeTreeAttributionOptions;
-    // (undocumented)
-    catchUpBlobName?: string;
-    mergeTreeEnableObliterate?: boolean;
-    mergeTreeEnableObliterateReconnect?: boolean;
-    mergeTreeEnableSidedObliterate?: boolean;
-    mergeTreeReferencesCanSlideToEndpoint?: boolean;
-    // (undocumented)
-    mergeTreeSnapshotChunkSize?: number;
-    newMergeTreeSnapshotFormat?: boolean;
-}
```

Comment:
Caution - this API was not marked `@deprecated`. Removable because no longer transitively accessible?